opal/mca/common/ofi/common_ofi.c

```diff
@@ -87,19 +91,30 @@ OPAL_DECLSPEC int opal_common_ofi_init(void)
 {
```

Comment:
All functions that do interesting work should have a doxygen style header.

You should call out things like what this function is doing and why.  At some point, you should explicitly say that you're injecting a monitor into Libfabric so that Libfabric can use Open MPI's memory hooks as its memory monitor, rather than trying to use its own.

Reply:
I had a comment in the header file for the function, but it isn't doxygen style, I'll update it and give it more information